### ISSUE TITLE
Linting imports - unexpected behavior

### ISSUE THREAD

**Issue by cvn** [jgable/grunt-lesslint#35]
It appears that lesslint is linting imports by default now.

My config is like this:

```
grunt.config('lesslint',{
    less: {
        src: ['static/example.less']
    },
    options: {
        csslint: {
          csslintrc: '.csslintrc'
        }
    }
});
```

example.less is like this

```
@import 'include.less';
```

and include.less is like this

```
.error {
```

When I run `grunt lint` I get this error:

```
Linting 'example.less...OK
Error parsing example.less: missing closing `}`
```

When I fix the error in include.less, it lints fine.

I tried setting `options.includes: false`, and it had no effect.

The readme says that it should not lint imports by default, and I think this was accurate in previous versions. Can you confirm this behavior is unexpected?

Also, the error references the wrong file. It should report the sytax error in include.less, not example.less.

I'm running grunt-lesslint 1.1.2
